<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Thresholding</class>
 <widget class="QMainWindow" name="Thresholding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thresholding</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>201</width>
      <height>117</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_Type">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="btn_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="comboBox_type">
       <item>
        <property name="text">
         <string>Binary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Inverse Binary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Trunc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tozero</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Inverse Tozero</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="num_threshold">
       <property name="maximum">
        <number>254</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="num_maximum">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_Threhold">
       <property name="text">
        <string>Threshold</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_Maximum">
       <property name="text">
        <string>Maximum</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Thresholding</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
